libvast/CMakeLists.txt

```
@@ -47,8 +47,8 @@ endif ()
 # -- flatbuffers ---------------------------------------------------------------
 
 file(GLOB flatbuffers_schemas CONFIGURE_DEPENDS
-     "${CMAKE_CURRENT_SOURCE_DIR}/vast/fbs/*.fbs"
-     "${CMAKE_CURRENT_SOURCE_DIR}/vast/fbs/legacy/*.fbs")
+     "${CMAKE_CURRENT_SOURCE_DIR}/include/vast/fbs/*.fbs"
+     "${CMAKE_CURRENT_SOURCE_DIR}/include/vast/fbs/legacy/*.fbs")
```

We should move those to a more logical place while we're at it
```suggestion
     "${CMAKE_CURRENT_SOURCE_DIR}/fbs/*.fbs"
     "${CMAKE_CURRENT_SOURCE_DIR}/fbs/legacy/*.fbs")
```
I think we should probably also install them?